io/post_fv3.F90

```diff
@@ -2138,164 +2335,641 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
               do j=jsta,jend
                 do i=ista, iend
-                  pbotl(i,j) = arrayr42d(i,j)
-                  if( abs(arrayr42d(i,j)-fillValue) < small) pbotl(i,j) = spval
+                  buf2(i,j) = arrayr42d(i,j)
+                  if(abs(arrayr42d(i,j)-fillvalue)<small) buf2(i,j) = spval
                 enddo
               enddo
             endif
+            ! !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,buf,buf2,acsnow)
+             do j=jsta,jend
+               do i=ista, iend
+                 if(buf(i,j)<spval .and. buf2(i,j)<spval) then
+                   acsnow(i,j) = buf(i,j) + buf2(i,j)
+                 else
+                   acsnow(i,j) = spval
+                 endif
+               enddo
+             enddo
+             endif !FV3R
 
-            ! time averaged low cloud top temperature
-            if(trim(fieldname)=='tmp_avelct') then
-              !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,ttopl,arrayr42d,fillValue)
+            !sndepac
+            if(trim(fieldname)=='snacc_land') then
+              !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,buf,arrayr42d,fillvalue,spval)
               do j=jsta,jend
                 do i=ista, iend
-                  ttopl(i,j) = arrayr42d(i,j)
-                  if( abs(arrayr42d(i,j)-fillValue) < small) ttopl(i,j) = spval
+                  buf(i,j) = arrayr42d(i,j)
+                  if(abs(arrayr42d(i,j)-fillvalue)<small) buf(i,j) = spval
                 enddo
               enddo
             endif
-
-            ! time averaged middle cloud top pressure
-            if(trim(fieldname)=='pres_avemct') then
-              !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,ptopm,arrayr42d,fillValue)
+            if(trim(fieldname)=='snacc_ice') then
+              !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,buf2,arrayr42d,fillvalue,spval)
               do j=jsta,jend
                 do i=ista, iend
-                  ptopm(i,j) = arrayr42d(i,j)
-                  if( abs(arrayr42d(i,j)-fillValue) < small) ptopm(i,j) = spval
+                  buf2(i,j) = arrayr42d(i,j)
+                  if(abs(arrayr42d(i,j)-fillvalue)<small) buf2(i,j) = spval
                 enddo
               enddo
             endif
-            ! time averaged middle cloud bottom pressure
-            if(trim(fieldname)=='pres_avemcb') then
-              !$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,pbotm,arrayr42d,fillValue)
-              do j=jsta,jend
-                do i=ista, iend
-                  pbotm(i,j) = arrayr42d(i,j)
-                  if( abs(arrayr42d(i,j)-fillValue) < small) pbotm(i,j) = spval
-                enddo
+            !!$omp parallel do default(none) private(i,j) shared(jsta,jend,ista,iend,spval,buf,buf2,sndepac)
```

Comment:
Is `!!` intended here?

Reply:
@DusanJovic-NOAA Thanks for catching it. Will delete one !.